Library/Homebrew/description_cache_store.rb

```diff
@@ -34,9 +34,9 @@ def delete!(formula_name)
   #
   # @return [nil]
   def populate_if_empty!
+    return unless Homebrew.EnvConfig.trust_all?
```

Comment:
Do we want this for the early returns in this file?

```suggestion
    return unless Homebrew.EnvConfig.trust_all? || ARGV.include?("--all")
```

Or pass `args.all?` to this somehow?

Reply:
@Rylan12 No. It can be initialised by too many things including `brew update` background updates. It makes more sense to entirely guard this on the environment variable.